content/PoolsPage.tsx

```diff
@@ -24,15 +23,15 @@ function Pool() {
   const [stakedTokens, setStakedTokens] = useState(BigNumber.from(0));
   const { t } = useTranslation();
 
-  const { walletAddress } = Connector.useContainer();
+  const { walletAddress, synthetixjs } = Connector.useContainer();
 
-  const snx = synthetix({ networkId: NetworkIdByName['mainnet-ovm'], useOvm: true });
```

Comment:
I'm not sure this page is active. But here we're are changing behaviour. Before syntehtix was always initialised with optimism as network. Where as now it will be initialised with whatever network you are connected too

Reply:
This contract is only on optimism mainnet, so in useGetUniswapStakingRewardsAPY we throw an error, but maybe for clarity's sake should just hardcode for optimism here.

Reply:
Yea I think it's fine since it also currently isn't used.